PhysicsTools/NanoAOD/plugins/BJetEnergyRegressionVarProducer.cc

```diff
@@ -132,16 +132,13 @@ template <typename T>
 void BJetEnergyRegressionVarProducer<T>::produce(edm::StreamID streamID,
```

Comment:
This file has quite several duplicate header includes: could you please profit and clean them up?